commands/videostats/index.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 	Flags: ["mention","pipe","whitelist"],
 	Whitelist_Response: "Video statistics are only available in Cytube rooms.",
 	Static_Data: null,
-	Code: async (extra, link) => {
+	Code: (async function videostats (extra, link) {
```

Comment:
rename the `extra` parameter to `context`, as that is the name we use in the current style.